.github/workflows/rubocop.yml

```
@@ -0,0 +1,39 @@
+name: "Rubocop"
```

```suggestion
name: "RuboCop"
```
Correct capitalization again.